### PR TITLE
Update to Zope  5 and Python 3

### LINKED ISSUE #1: Zope 4 compatibility
To make this product compatible with Zope 4, just change an import in the components.py file:

#from zope.location.interfaces import IPossibleSite
from zope.component.interfaces import IPossibleSite

tested in Zope 4.5.3 x py2 and py3

### PR DESCRIPTION
**Pull request by icemac**
While porting packages to GitHub Actions I found this PR.

Is it still valid? Is it worth to put more effort into it?
If not, I'd like to archive this repository as it is missing a Python 3 port of the code and so it is probably be unused nowadays.

cc: @goschtl @dhavlik @openlegis-br

This package is a dependency of a customer, so we completed this PR.

Fixes #1.

### CODE REVIEW

**Comment by openlegis-br** [comment]
can be important to port old projects, if there is no substitute in py3

Em qui, 25 de mar de 2021 04:24, Michael Howitz ***@***.***>
escreveu:

**Comment by sallner** [comment]
I used the zopefoundation/meta config tool to harmonize the testing.

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@ Changelog
   version comparison in ``setuptools 8.0``.
   [maurits]
 
+- Add support for Python 3.7, 3.8, 3.9, 3.10, 3.11.
+
+- Add support for Zope >= 4.
```
Comment (icemac) [inline docs/HISTORY.txt:13]:
```suggestion
- Add support for Zope >= 4.

- Drop support for Zope 2.13.
```

File: src/five/grok/components.py [hunk 2]
```diff
@@ -188,11 +187,12 @@ def __getitem__(self, key):
         # XXX Need Zope 2 __getitem__
```
Comment (icemac) [inline src/five/grok/components.py:187]:
```suggestion
        # XXX Need Zope 4 __getitem__
```

**Review by sallner — APPROVED** [review]

**Pull request merged**